_data/other.yml

```diff
@@ -222,6 +222,13 @@ websites:
       software: Yes
       doc: https://www.findmyshift.com/help/what-is-two-factor-authentication
 
+    - name: Firefox
+      url: https://accounts.firefox.com/settings/two_step_authentication
```

Comment:
This url should point either to the main firefox.com or at least accounts.firefox.com, but not settings page.

Reply:
Okay, thanks, I will do so.